docs/api/public_php_api_search.md

```diff
@@ -450,8 +450,71 @@ use eZ\Publish\API\Repository\Values\Content\Query;
     Make sure that you set the Criterion on the `filter` property.
     It is impossible to use the `query` property, because the search in trash operation filters the database instead of querying.
 
+## Aggregation
+
+!!! caution "Feature support"
+
+    Aggregation is only available in the Solr and Elasticsearch search engines.
+
+Aggregation enables you to find the count of search results for each Aggregation type.
+
+To do this, you need to make use of the query's `$aggregations` property:
+
+``` php
+$query->aggregations[] = new ContentTypeTermAggregation('content_type');
+```
+
+The name of the aggregation must be unique in the given query.
```

Comment:
Is "content_type" the name of the aggregation? Could we use "<aggregation_name>" and explain that it can be, for example `content_type`?

Reply:
The whole Public PHP API doc section is based on actual examples that should be directly usable, the same name is used in examples further down.